<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FncDialog</class>
 <widget class="QDialog" name="FncDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Function Type Selection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../src/ressources/PLCEdit.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <widget class="QWidget" name="horizontalLayout">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>324</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Fnc_HBoxLayout_okcancel">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QPushButton" name="Fnc_okButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Fnc_cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTreeWidget" name="Fnc_treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>280</width>
     <height>271</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>280</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideRight</enum>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <column>
    <property name="text">
     <string> Function Types</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>Binary</string>
    </property>
    <property name="icon">
     <iconset resource="../src/ressources/PLCEdit.qrc">
      <normaloff>:/images/CVtypedef_16x16.png</normaloff>:/images/CVtypedef_16x16.png</iconset>
    </property>
    <item>
     <property name="text">
      <string>BOOL</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BOOL_R_EDGE</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BOOL_F_EDGE</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BYTE</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WORD</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DWORD</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Date and Time</string>
    </property>
    <property name="icon">
     <iconset resource="../src/ressources/PLCEdit.qrc">
      <normaloff>:/images/CVtypedef_16x16.png</normaloff>:/images/CVtypedef_16x16.png</iconset>
    </property>
    <item>
     <property name="text">
      <string>DATE</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DATE_AND_TIME</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DT</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TIME</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TIME_OF_DAY</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TOD</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Integer</string>
    </property>
    <property name="icon">
     <iconset resource="../src/ressources/PLCEdit.qrc">
      <normaloff>:/images/CVtypedef_16x16.png</normaloff>:/images/CVtypedef_16x16.png</iconset>
    </property>
    <item>
     <property name="text">
      <string>SINT</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>USINT</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>INT</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UINT</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DINT</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UDINT</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>REAL</string>
    </property>
    <property name="icon">
     <iconset resource="../src/ressources/PLCEdit.qrc">
      <normaloff>:/images/CVtypedef_16x16.png</normaloff>:/images/CVtypedef_16x16.png</iconset>
    </property>
    <item>
     <property name="text">
      <string>REAL</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LREAL</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Strings</string>
    </property>
    <property name="icon">
     <iconset resource="../src/ressources/PLCEdit.qrc">
      <normaloff>:/images/CVtypedef_16x16.png</normaloff>:/images/CVtypedef_16x16.png</iconset>
    </property>
    <item>
     <property name="text">
      <string>STRING</string>
     </property>
     <property name="icon">
      <iconset resource="../src/ressources/PLCEdit.qrc">
       <normaloff>:/images/FcFileAdd_16x16.png</normaloff>:/images/FcFileAdd_16x16.png</iconset>
     </property>
    </item>
   </item>
  </widget>
  <widget class="QLineEdit" name="Fnc_lineEdit_userDefinedType">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>290</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>151</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Fnc_label_userDefinedType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>292</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>131</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>131</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>User defined Type:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Fnc_cancelButton</tabstop>
  <tabstop>Fnc_okButton</tabstop>
 </tabstops>
 <resources>
  <include location="../src/ressources/PLCEdit.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Fnc_cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>FncDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
